src/context/ChainApiContext.tsx

```diff
@@ -30,7 +29,6 @@ export function ChainApiContextProvider({children}: {children: React.ReactNode})
       return;
     }
 
-    queryClient.clear();
```

Comment:
`queryClient` is not used for anything else, can you remove the usage of `useQueryClient` as well?